release-please-config.json

```
@@ -0,0 +1,23 @@
+{
+  "bootstrap-sha": "9903e0c129683979234f287d59f7468ca5cd7341",
+  "exclude-paths": [
+    ".git",
+    ".idea",
+    ".github",
+    ".vscode"
+  ],
+  "release-type": "dotnet",
```

Strategy does not exist at https://github.com/googleapis/release-please/blob/72b0ab360c3d6635397e8b02f4d3f9f53932e23c/docs/customizing.md#strategy-language-types-supported

```suggestion
  "release-type": "dotnet-yoshi",
```